grunt/tasks/jest.js

```diff
@@ -1,84 +1,17 @@
 'use strict';
 
-var async = require('async');
 var fs = require('fs');
-var glob = require('glob');
 var grunt = require('grunt');
 var path = require('path');
 
-var rootPath = path.resolve('.');
```

Comment:
Jest has this annoying mode where you can either use the default where it runs coverage on all the files that are required by tests (the current mode) or you can manually specify all the files/folders you want to whitelist and blacklist.

My thinking was that outside of this thirty party component file, the rest is actually good. When I tried doing it on everything then it reported a bunch of useless stuff.

By the way, we don't need all that logic anymore, jest has a setting `collectCoverageFrom` in package.json that lets us configure it: https://github.com/facebook/jest/blob/master/package.json#L47

If you want to go through all the files and make it list only the things we care about, let me know.